libnavigation-util/src/main/java/com/mapbox/navigation/utils/time/Time.kt

```diff
@@ -1,4 +1,4 @@
-package com.mapbox.services.android.navigation.v5.internal.utils
+package com.mapbox.navigation.util
```

Comment:
Same as in https://github.com/mapbox/mapbox-navigation-android/pull/2240#discussion_r344967234